pkgs/development/haskell-modules/configuration-ghc-9.6.x.nix

```
@@ -142,11 +144,27 @@ self: super: {
   generically = jailbreakWhileRevision 0 super.generically;
   # Revision 3 lifts offending template-haskell bound
   hackage-security = jailbreakWhileRevision 2 super.hackage-security;
+  # Revision 6 lifts offending base bound
+  parallel = jailbreakWhileRevision 5 super.parallel;
 
   #
   # Compilation failure workarounds
   #
 
+  # Add missing Functor instance for Tuple2
+  # https://github.com/haskell-foundation/foundation/pull/572
+  foundation = pkgs.haskell.lib.overrideCabal (super.foundation) {
+    patches = [
+      (pkgs.fetchpatch {
+        name = "foundation-pr-572.patch";
+        url =
+          "https://patch-diff.githubusercontent.com/raw/haskell-foundation/foundation/pull/572.patch";
```

Usually better to use the content-addressed commit instead of the mutable PR reference, so the `sha256` is never invalidated.


```suggestion
          "https://github.com/haskell-foundation/foundation/commit/d3136f4bb8b69e273535352620e53f2196941b35.patch";
```